### PR TITLE
4.3.2 release notes

### PR DESCRIPTION
**Pull request by MichaelBaj**

### CODE REVIEW

File: docs/release_notes/128t_release_notes_4.3.2.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+  Until the system is upgraded to 4.3.2, this issue can be mitigated by using the PCLI versions of the same command.
+
+- **I95-33759** GUI DHCP Client Lease table reports no leases while DHCP server has active client leases.
+  Until the system is upgraded to 4.3.2, this issue can be mitigated by using the PCLI version of the same command: `show network-interface application node <node-name> name <interface-name>`
+
+- **I95-33789** Forwarding core count incorrectly reported in GUI under Router -> Node -> Platform Information
+  Until the system is upgraded to 4.3.2, this issue can be mitigated by using the PCLI version of the same command: `show platform cpu `
+
+- **I95-33938** REST API missing documentation reference to command to delete configuration backups.  The missing, and supported, API is  `DELETE /api/v1/config/export/:filename`
```
Comment (jeffmart) [inline docs/release_notes/128t_release_notes_4.3.2.md:24]: consider rewording (to to for):

REST API missing documentation reference for the command to

Also should the comma be removed here?

The missing, and supported,

File: docs/release_notes/128t_release_notes_4.3.2.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  ```
+
+  _**Conditions:**_ Automated Provisioner performs a software version query 
+  
+- **I95-35477** Interface never becomes active when `shared-physical-address` is configured to be the same as the physical MAC
+
+- **I95-34645** Swagger API for "clone" and "move" operations are incorrect.  They are `/config/{configStore}/authority/district/{district}/clone` when they should be `/config/{configStore}/authority/district/clone`
+
+- **I95-34653** SNMP IF-MIB does not display correctly when a non-forwarding management interface is present
+
```
Comment (jeffmart) [inline docs/release_notes/128t_release_notes_4.3.2.md:84]: entry needed here:

- **I95-33845** security package updates

File: docs/release_notes/128t_release_notes_4.3.2.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  
+  ```
+  /usr/lib/128technology/par/connect128t.par/subpar/runtime/support.py:117: UserWarning: Unable to extract to requested directory '/usr/lib/128technology/unpar/connect128t', falling back to tmp dir
+  ```
+  There is no impact to the functionality of `connect128t`
+
+- **I95-34068** SVR sessions fail to establish due to waypoint allocation failures after HA node failover.
+
+  _**Symptom:**_ The following warning log is generated:
+
```
Comment (jeffmart) [inline docs/release_notes/128t_release_notes_4.3.2.md:46]: similar warning message provided as the IP addresses may be different. 

This also bring up an interesting aspect. We need to be very careful when using addresses and make sure that they are all private addresses and nothing public. General rule.

File: docs/release_notes/128t_release_notes_4.3.2.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+- **I95-34371** Large FIBs can take excess of 30 seconds to load, causing the highway process to fault and restart
+
+  _**Symptom:**_ System would appear to take a long time to load its FIB
+
+  _**Conditions:**_ System running at TRACE log level may induce this problem
+
+  Until the system is upgraded to 4.3.2, this issue can be mitigated by ensuring the system is configured at log level INFO, and if necessary, reduce the size of the service configuration.
+
+- **I95-34397** ARP caching and reverse mac learning is not triggered when ARP response takes longer than 1 second
```
Comment (ptimmons) [inline docs/release_notes/128t_release_notes_4.3.2.md:65]:
```suggestion
- **I95-34397** ARP caching and reverse MAC learning is not triggered when an ARP response takes longer than 1 second
```

File: docs/release_notes/128t_release_notes_4.3.2.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+- **I95-34421** Lenovo onboard Intel X722 NIC is not recognized by 128T and can not used as a forwarding interface
```
Comment (ptimmons) [inline docs/release_notes/128t_release_notes_4.3.2.md:67]:
```suggestion
- **I95-34421** Lenovo onboard Intel X722 NIC is not recognized by 128T and cannot be used as a forwarding interface
```

File: docs/release_notes/128t_release_notes_4.3.2.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+  /usr/lib/128technology/par/connect128t.par/subpar/runtime/support.py:117: UserWarning: Unable to extract to requested directory '/usr/lib/128technology/unpar/connect128t', falling back to tmp dir
+  ```
+  There is no impact to the functionality of `connect128t`
+
+- **I95-34068** SVR sessions fail to establish due to waypoint allocation failures after HA node failover.
+
+  _**Symptom:**_ The following warning log is generated:
+  
+  ```
+Mar 03 09:25:10.813 [HWMC| – ] WARN (icmpManager ) Base Exception: failed to allocate ports for WayPoint; intf=5.0; local=172.27.233.47; remote=10.61.55.109
```
Comment (ptimmons) [inline docs/release_notes/128t_release_notes_4.3.2.md:48]:
```suggestion
Mar 03 09:25:10.813 [HWMC| – ] WARN (icmpManager ) Base Exception: failed to allocate ports for WayPoint; intf=5.0; local=192.0.2.100; remote=198.51.100.128
```

File: docs/release_notes/128t_release_notes_4.3.2.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+
+  _**Symptom:**_ Log messages appear in salt minion file
+
+  ```
+2020-03-25 17:59:37,893 [salt.loader.12.230.70.246.int.module.cmdmod][ERROR ][18140] stdout: package centos-release-yum4 is not installed
+  ```
+
+  _**Conditions:**_ Automated Provisioner performs a software version query 
+  
+- **I95-35477** Interface never becomes active when `shared-physical-address` is configured to be the same as the physical MAC
```
Comment (ptimmons) [inline docs/release_notes/128t_release_notes_4.3.2.md:79]:
```suggestion
- **I95-35477** Interface never becomes active when `shared-phys-address` is configured to be the same as the physical MAC
```

**Pull request closed without merging**